diff.py

```diff
@@ -2188,7 +2205,16 @@ def diff_sameline(old: str, new: str) -> None:
         if ignore_last_field:
             newfields = newfields[:-1]
             oldfields = oldfields[:-1]
+        else:
+            ### If the last field has a register suffix, i.e. "0x38(r7)"  we split that part out to make it a seperate field
```

Comment:
```suggestion
            # If the last field has a register suffix, i.e. "0x38(r7)"  we split that part out to make it a separate field
```

Reply:
also splitting on "(" seems dangerous for mips, which uses syntax like `%lo(sym)` for relocs; for address modes you thus get something like `%lo(sym)($reg)`. It would be nice if we could store symbols away in a separate field and not have to re-parse them here

Reply:
I could be wrong, but I looked though a bunch of examples of mips asm, and I don't think it causes any issues.
Actually, in some cases it may lead to slightly more accurate scoring, if there are multiple diffs in the line.